doc/glossary.md

```diff
@@ -0,0 +1,313 @@
+# Glossary
+
+#### ADC
+
+Analog-to-Digital Converter.
+
+#### AES
+
+Advanced Encryption Standard.
+
+#### Airgapped
+
+Isolated from the surrounding environment, particularly networks or the operating system.
+
+#### AON
+
+Always On.
+
+#### ASM
+
+Assembly code.
+
+#### Attestation
+
+The process of a device taking some measurement and checking it matches what is expected.
+
+#### CIP
+
+Comportable (hardware) Intellectual Property.
+
+#### Comportable
+
+See the [comportability specification](./contributing/hw/comportability/README.md).
+
+#### CRC
+
+Cyclic Redundancy Check.
+
+#### CSR
+
+Control/Status Register.
+
+#### CSRNG
+
+Cryptographically Secure Random Number Generator.
+
+#### CTAP
+
+Client to Authenticator Protocol.
+
+A protocol for authentication devices to communicate with a host computer.
+
+#### CW310
+
+[ChipWhisperer CW310](https://rtfm.newae.com/Targets/CW310%20Bergen%20Board/) "Bergen" FPGA board.
+
+#### CW340
+
+[ChipWhisperer CW340](https://github.com/newaetech/cw340-luna-board) "Luna" FPGA baseboard.
+
+#### Darjeeling
+
+A work-in-progress integrated OpenTitan top level.
+
+#### DD
+
+Digital Design.
+
+#### DV
+
+Design Verification.
+
+#### DVsim
```

Comment:
We almost-exclusively use this acronym as lower-case "dvsim" throughout the codebase.
I think it's probably worth doing the same here?